openapi/components/schemas/Websites/WebsiteSettings.yaml

```
@@ -26,3 +26,33 @@ properties:
             nullable: true
             items:
               type: string
+  shippo:
+    type: object
+    description: Shippo settings.
+    properties:
+      enabled:
+        type: boolean
+        description: Specifies whether Shippo is enabled.
+      credentialHash:
+        type: string
+        description: Shippo credential hash.
+      filter:
+        type: string
+        description: Shippo filter.
+      weight:
+        type: string
+        description: Shippo weight.
+      weightUnit:
+        type: string
+        enum:
+          - g
+          - oz
+          - lb
+          - kg
+        description: Shippo weightUnit.
```

This is not an enum, but a template that evaluates to the enum.
```suggestion
      weightUnit:
        type: string
        description: Shippo weight unit. Supports mustache templates within the invoice context. Must evaluate to a valid Shippo weight unit: `g`, `oz`, `lb` or `kg`.
```

---

Updated, thanks